starter-code/css/main.css

```diff
@@ -6,3 +6,172 @@ Green: #00B172
 White: #FFF
 
 */
+
+*{
+    margin : 0;
+    padding: 0;
+}
+body {
+    font-family: Helvetica;
+    font-weight: lighter;
+    font-size: 20px; 
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position :fixed;
+    background-color: white;
+    height : 100px;
+
+}
+
+nav ul{
+   display: flex;
+    text-decoration: none;
+    list-style-type: none;
+    flex-direction: row;
+    margin : 0 30px;
+    width : 300px;
+    justify-content: space-between;
+    
+}
+
+nav ul li{
+    margin : 0 10px;
+    font-size: 1rem;
+}
+
+nav img {
+    width: 20%;
+    margin: 0 0 0 50px;
+
+}
+
+nav img:hover{
+    animation : 2s bounce;
+}
+
+.header{
+    background-image: url("../images/landing.jpg");
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 90vh;
+    width: 100%;
+    color: white;
+}
+
+section h2{
+    margin : 0 auto;
+    margin-top : 40px;
+    width: 50%;
+    text-decoration: underline  #00B172;
+    padding: 50px;
+    text-align: center;
+}
+.features {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+}
+.header h1{
+    font-weight: bolder;
+    font-size: 5rem;
+    padding: 30px;
+    margin: 10px auto;
+}
+
+.header p{
+    font-size: 1.5rem;
+    padding: 20px;
+    margin: 10px auto;
+    text-align: center;
+}
+
+.col {
+    width: 30%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height : 300px;
+
+}
+.col img{
+    height: 40%;
+    margin: 10%;
+}
+
+.col h3{
+    color: #00B172;
+    font-size: 1.5rem;
+    font-weight: bold;
+}
+.col p{
+    margin : 5% 0;
+    font-size: 1rem;
+    padding: 10px;
+    text-align: center;
+}
+
+
+.big-green-div{
+    background-color: #00B172;
```

Comment:
you could have added the white spotify logo as a background  to this div

```
  background-image: url(../images/spotify-icon-white.png);
  background-repeat: no-repeat;
  background-position: center center;
  background-size: 10%;

```

It would makes it easier to position the text and phone image